Makefile

```diff
@@ -276,11 +258,10 @@ snapcraft-local.yaml: snapcraft-local.yaml.in
 # To avoid unintended conflicts with file names, always add to .PHONY
 # unless there is a reason not to.
 # https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html
-.PHONY: install install_debug dist clean \
-draw_deps test test_cli test_unit \
-test_cover lint benchmark devdoc_init devdoc devdoc_save devdoc_update \
-build-linux build-docker-gaiadnode localnet-start localnet-stop \
-format check-ledger test_sim_gaia_nondeterminism test_sim_modules test_sim_gaia_fast \
+.PHONY: build dist clean draw_deps test test_unit test_cover lint \
+benchmark devdoc_init devdoc devdoc_save devdoc_update runsim \
+build-docker-gaiadnode localnet-start localnet-stop \
```

Comment:
`build-docker-gaiadnode` should be renamed

Reply:
there shouldn't be a docker target at all, right?

Reply:
good question, I guess not ? @sabau @alessio

Reply:
Nice catch folks! Removed a bunch of gaia-related things